### PR TITLE
Vendorruntimetools

### PR DESCRIPTION
**Pull request by baude**
Update runtime-tools and buildah to account for the the generator functions and Darwin compilability.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@mheon @wking @umohnani8 @TomSweeneyRedHat PTAL

**Comment by mheon** [comment]
LGTM pending tests

File: pkg/spec/spec.go [hunk 1]
```diff
@@ -21,7 +21,10 @@ const cpuPeriod = 100000
 // CreateConfigToOCISpec parses information needed to create a container into an OCI runtime spec
 func CreateConfigToOCISpec(config *CreateConfig) (*spec.Spec, error) { //nolint
 	cgroupPerm := "ro"
-	g := generate.New()
+	g, err := generate.New("linux")
```
Comment (wking) [inline pkg/spec/spec.go]: The rest of `CreateConfigToOCISpec` is pretty Linux-specific, so I'm fine with this being hard-coded here.  But `CreateConfigToOCISpec` is so opinionated, I'm surprised you need to seed it at all.  Do you know what parts of this initial config survive the subsequent edits to be returned to the caller?  You might just want to build the `*spec.Spec` object directly.

**Comment by umohnani8** [comment]
LGTM, but tests are unhappy

File: libpod/common_test.go [hunk 2]
```diff
@@ -75,7 +75,10 @@ func getTestContainer(id, name, locksDir string) (*Container, error) {
 		valid: true,
 	}
 
-	g := generate.New()
+	g, err := generate.New("linux")
+	if err != nil {
+		return nil, err
+	}
 	ctr.config.Spec = g.Spec()
```
Comment (wking) [inline libpod/common_test.go]: This should have `g.Spec()` -> `g.Config` (and similarly in `CreateConfigToOCISpec`).

**Comment by baude** [comment]
@rh-atomic-bot r+

**Comment by wking** [comment]
Hmm, looks like this PR also forgot to bump `vendor.conf`.  @baude, did you want to file a fixup with the dependency commits you used?  Or should we deal with this if/when I have time to get back to #474?

File: libpod/common_test.go (hunk 2 above)
Comment (wking) [reply libpod/common_test.go]: > This should have `g.Spec()` -> `g.Config` (and similarly in `CreateConfigToOCISpec`).

I've filed #1008 for this.

**Pull request closed without merging**